packages/@okta/vuepress-site/docs/guides/sign-into-spa/define-callback/react/define-route.md

```diff
@@ -17,3 +17,7 @@ const App = () => {
 
 export default App;
 ```
+
+Our examples use `login/callback` as a default route path, but you can change this. The route path is used in the next step.
```

Comment:
```suggestion
These examples use `login/callback` as a default route path. The route path is used in the next step.
```
If you feel it's important to state that the value for default route path can be changed, you can add `", use a value appropriate to your application"`; I don't feel strongly that this would be better (or worse).